quickstart/aws/tutorial-rest-api.md

```diff
@@ -23,8 +23,9 @@ concepts to explore additional containers, serverless, and infrastructure tutori
 Let's use the Pulumi CLI to initialize a new project:
 
 ```
-pulumi new hello-aws-javascript --dir ahoy-pulumi
+mkdir ahoy-pulumi 
 cd ahoy-pulumi
+pulumi new hello-aws-javascript
```

Comment:
Let's put the `mkdir` and `cd` on the same line:

```
mkdir ahoy-pulumi && cd ahoy-pulumi
pulumi new hello-aws-javascript
```

#964 tracks applying this consistently everywhere.